### ISSUE TITLE
Why is tr and tcl both represented by two channels? Is it ok to use one channel?

### ISSUE THREAD

**Issue by ustczhouyu** [princewang1994/TextSnake.pytorch#39]
Why is tr and tcl both represented by two channels? Is it ok to use one channel?

**Comment by spoorgholi74** [comment]
Hi, thanks for the code.
I am also confused why 2 channels are used for the TCL & TR?
it will be great if you can briefly explain or refer me to where it is explained in paper.

**Comment by princevandev** [comment]
@spoorgholi74 , without special resons. if you're using sigmoid activate function, one channel is required. Two-channel feature map is set to be compatible with softmax activate function and cross-entropy loss, use one channel if you prefer it :)
